src/frontend/templates/home.html

```diff
@@ -1,80 +1,182 @@
   <div class="container-fluid">
-    <div class="row">
-
-      <!-- The image on the left of the home page, displayed on larger screens. -->
-      <!--<div class="col-4 d-none d-lg-block home-desktop-left-image"></div>-->
-      <!-- @TODO: removed temporarily. When uncommenting, also replace below div with this -->
-      <!--<div class="col-12 col-lg-8">-->
-
-      <div class="col-12 col-lg-12 px-10-percent">
-
-        <div class="row hot-products-row px-xl-6">
-
-          <div class="col-12">
-            <h3>Hot Products</h3>
-          </div>
-
-          {{ range $.products }}
-          <div class="col-md-4 hot-product-card">
-            <a href="{{ $.baseUrl }}/product/{{.Item.Id}}">
-              <img loading="lazy" src="{{ $.baseUrl }}{{.Item.Picture}}">
-              <div class="hot-product-card-img-overlay"></div>
-            </a>
-            <div>
-              <div class="hot-product-card-name">{{ .Item.Name }}</div>
-              <div class="hot-product-card-price">{{ renderMoney .Price }}</div>
+    <div class="row hot-products-row px-xl-6">
+      {{ range $.products }}
+      <div class="col-md-4 hot-product-card">
+        <a href="{{ $.baseUrl }}/product/{{.Item.Id}}">
+          <img loading="lazy" src="{{ $.baseUrl }}{{.Item.Picture}}">
+        </a>
+        <div>
+          <div class="hot-product-card-name">{{ .Item.Name }}</div>
+          <div class="hot-product-card-price">{{ renderMoney .Price }}</div>
+          <!-- Add to Wishlist Button -->
+          <div class="dynamic-wishlist-button" style="margin-top: 10px;">
+            <div class="container" style="position: relative;">
+              <button class="wishlist-toggle-button" data-product-id="{{ .Item.Id }}">
+                ♥ Add to Wishlist
+              </button>
+              <div class="dropdown">
+                <div class="dropdown-list"></div>
+              </div>
             </div>
           </div>
-          {{ end }}
-
-        </div>
-
-        <!-- Footer for larger screens. -->
-        <div class="row d-none d-lg-block home-desktop-footer-row">
-          <div class="col-12 p-0">
-            {{ template "footer" . }}
-          </div>
         </div>
-
       </div>
-
+      {{ end }}
     </div>
   </div>
-
 </main>
 
-<!-- Footer for smaller screens. -->
-<div class="d-lg-none">
-  {{ template "footer" . }}
-</div>
-
+<script>
+  document.addEventListener("DOMContentLoaded", () => {
+    const wishlistButton = document.querySelector(".wishlist-button");
+    const dropdown = document.querySelector(".dropdown");
+    const dropdownList = document.querySelector(".dropdown-list");
+    const addButton = document.querySelector(".add-button");
+    const toggleButtons = document.querySelectorAll(".wishlist-toggle-button");
+
+    let wishlists = JSON.parse(localStorage.getItem("wishlists")) || ["My Wishlist"];
+    let productWishlists = JSON.parse(localStorage.getItem("productWishlists")) || {};
```

Comment:
localStorage could be not supported or disabled in some browsers but I would leave it for now and come back if it makes trouble 😆